File: java/2 - fila/Banco.java
public class Banco {
  public static void main(String args[]) {
    Fila f;
    int e;
    f = new Fila();

    f.insert(176);
    f.insert(914);
    f.insert(12);
    f.insert(1817);
    f.insert(100);

    System.out.println("Bem vindos ao banco Izibank! Servimos bem, para servirmos sempre!");
    System.out.println("Atendimento-------------");

    while (!f.isEmpty()) {
      e = f.remove();
      System.out.println("Senha a ser atendida no guiche: " + e);
    }
  }
}

File: c/2 - fila/banco.c
#include <stdio.h>
#include "fila.c"

int main(int argc, char *argv[])
{

  struct Fila f;
  int elemento;
  create(&f);

  insert(&f, 913);
  insert(&f, 914);
  insert(&f, 915);
  insert(&f, 916);
  insert(&f, 917);
  insert(&f, 918);

  printf("Bem vindos ao isibank! Aqui seu dinheiro rende mais\n");
  printf("Atendimento ---------------------\n");

  while (!isEmpty(f))
  {
    elemento = retirar(&f);
    printf("Numero de senha a ser atendida: %d\n", elemento);
  }

  return 0;
}